docs/installation/install_epinio_cli.md

```diff
@@ -68,8 +64,8 @@ The following commands were tested using cosign version 2.1.1.
 
 ### Verify File Checksum Signature
 
-Instead of signing all release assets, Epinio signs a checksums file containing the different
-release assets checksums. You can download/copy the three files 'epinio_1.9.0_checksums.txt.pem',
+Instead of signing all release assets, Epinio signs a file containing checksums for the release assets.
+You can download/copy the three files 'epinio_1.9.0_checksums.txt.pem',
```

Comment:
Questions:
Would we want to add colon after 'three files'?
Do we want checksum files between single quotes or backticks to be shown clearer?

Reply:
Yes, thanks. I'll do that.